.github/workflows/main.yml

```diff
@@ -1,13 +1,43 @@
 # Run test file with command:
 #   act pull_request -e testdata/act/pull-request.json
 
-name: Test find flags
+name: Test and Generate Docs
+
 on: pull_request
 
 jobs:
-  find-feature-flags:
+  Run_Go_Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
Use the sha instead of the version number

Reply:
Whats the reason for using the SHA? All the docs use v4

Reply:
Security requirement

Reply:
Basically a tag/branch can change, a sha not so much (or not as easy)

Reply:
makes sense, ty